virttest/qemu_storage.py

```diff
@@ -345,10 +346,10 @@ def snapshot_del(self, blkdebug_cfg=""):
                                        " parameters")
         if blkdebug_cfg:
             cmd += " blkdebug:%s:%s" % (blkdebug_cfg, self.image_filename)
+            return process.run(cmd, ignore_status=True)
```

Comment:
But you try to catch CmdError in autotest/tp-qemu#654
try:
     cmd_result = image_io.snapshot_del(blkdebug_cfg=blkdebug_cfg)
except error.CmdError, err:
     ....